catalog/app/containers/Bucket/PackageTree.js

```diff
@@ -189,16 +190,16 @@ function PkgCode({ data, bucket, name, revision, path }) {
           contents: dedent`
             import quilt3
             # browse
-            quilt3.Package.browse("${name}"${hashPy}, registry="s3://${bucket}")
+            p = quilt3.Package.browse("${name}"${hashPy}, registry="s3://${bucket}")
             # download
-            quilt3.Package.install("${nameWithPath}"${hashPy}, registry="s3://${bucket}", dest=".")
+            p = quilt3.Package.install("${name}"${pathPy}${hashPy}, registry="s3://${bucket}", dest=".")
```

Comment:
Maybe we should use a different python variable name here, to make it safer to copy-paste and get both variables in REPL:

```
l = Package.browse()
p = Package.install()
```

Reply:
Good catch. I think it's OK to use `p =` for both, since users will likely only do one of them and they both return the a Package object.